week-2/mandatory/4-sorting-algorithm.js

```diff
@@ -0,0 +1,54 @@
+/*
+At the start of the course, you worked in teams to sort your team members, labelled by
+numbers, in ascending or descending order.
+Today, you will be applying the sorting algorithm you used in that exercise in code! 
+Create a function called sortAges which:
+- takes an array of mixed data types as input
+- removes any non-number data types without using the built-in javascript filter method
+- returns an array of sorted ages in ascending order (without using the built-in javascript
+sort method 😎)
+You don't have to worry about making this algorithm work fast! The idea is to get you to
+"think" like a computer and practice your knowledge of basic JavaScript.
+*/
+
+function sortAges(arr) {
+
```

Comment:
Seems like you've missed this one. It would be great if you still gave it a shot! It's a good exercise to implement some of these algorithms.